### ISSUE TITLE
XMI Serializer creates invalid XMI

### ISSUE THREAD

**Issue by Daedo** [dkpro/dkpro-cassis#159]
**Describe the bug**
Running Deserialization & Serialization on some XMI documents creates multiple elements with the same xmi id.

**To Reproduce**
1. Have an XMI with a feature structure with the xmi id 1 and the Sofa at xmi id 2 (sofa num at 1).

```
...
<cas:Sofa mimeType="text" sofaID="_InitialView" sofaNum="1" sofaString="..." xmi:id="2"/>
<type:someFeature xmi:id="1"/>
```

2. Deserialize and Serialize:
```
typesystem = load_typesystem(ts)
cas = load_cas_from_xmi(xmi, typesystem=typesystem)
xmi = cas.to_xmi(pretty_print=True)
```
3. Now both the Sofa and the feature structure have xmi id 1. This can create errors when trying to deserialize using UIMA or when trying to resolve references. 
```
...
<cas:Sofa mimeType="text" sofaID="_InitialView" sofaNum="1" sofaString="..." xmi:id="1"/>
<type:someFeature [...] xmi:id="1"/>
```

[Example Files](https://github.com/dkpro/dkpro-cassis/files/6029538/data.zip)

**Expected behavior**
Deserialization and Serialization should not affect any id.

**Please complete the following information:**
 - Version: 0.2.10.dev0 
 - OS: OS X
 
**Additional context**
I'm building an application that requires sending serialized CAS between a Python client and UIMA server. I noticed that sometimes my python client was sending invalid XMI even if the client wasn't tasked to do anything but deserialize and serialize.

**Comment by jcklie** [comment]
Thank you for reporting. 0.2.10.dev0 is a pretty old version, can you try 0.5.1?

**Comment by Daedo** [comment]
Thank you, I've seen it was fixed in 0.5.1.
